plugins/org.jboss.tools.openshift.core/src/org/jboss/tools/openshift/core/IRouteChooser.java

```diff
@@ -20,6 +20,17 @@
 public interface IRouteChooser {
 
 	public IRoute chooseRoute(List<IRoute> routes);
+
+	/**
+	 * Returns user's decision to remember the chosen route.
+	 * This method should be called after chooseRoute(List<IRoute).
+	 * 
+	 * @return
+	 */
+	default public boolean isRememberChoice() {
```

Comment:
Ok, let's remove 'public'.
My personal preference, too, always was to see 'public' written where it is implied, but Java specs say 'it is discouraged to redundantly specify the public and/or abstract modifier for a method declared in an interface'.